Array.h

```diff
@@ -224,6 +224,7 @@ class Array: public Vector
     void prepend_dim(int size, const string& name = "");
     void prepend_dim(D4Dimension *dim);
     void clear_all_dims();
+    void rename_dim(const string &oldName = "", const string &newName = "");
```

Comment:
Is there ever a time when oldName or newName will be ""? If not, lets no use these default values.